bootcamp/tutorials/integration/rag_with_langflow.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
<p> </p><p> </p><h3>Steps to replace astraDB with Milvus:</h3><ol><li>Remove existing cards of Vector Store. Click on two AstraDB cards marked red in the above image, and press&nbsp;<em>backspace</em>&nbsp;to delete them.</li><li>Click on the&nbsp;<strong>Vector Store</strong>&nbsp;option in the sidebar, chose Milvus and drag it into the canvas. Do this twice as we need 2 Milvus cards, one for storing the file processing workflow and one for search workflow.</li><li>Link the Milvus Modules to the rest of the components. See the image below for reference.</li><li>Configure the Milvus credentials for both Milvus modules. The simplest way is to use Milvus Lite by setting&nbsp;<strong>Connection URI</strong>&nbsp;to&nbsp;<code>milvus_demo.db</code>. If you have a Milvus server self-deployed or on Zilliz Cloud, set the&nbsp;<strong>Connection URI</strong> to server endpoint and&nbsp;<strong>Connection Password </strong>to token (for Milvus that's concatenated "&lt;username&gt;:&lt;password&gt;", for Zilliz Cloud it's API Key).</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>